src/module.ts

```diff
@@ -172,7 +172,7 @@ export const enum FeatureFlags {
   Atomics = 1 /* _BinaryenFeatureAtomics */,
   MutableGlobals = 2 /* _BinaryenFeatureMutableGlobals */,
   TruncSat = 4 /* _BinaryenFeatureNontrappingFPToInt */,
-  SIMD = 8 /* _BinaryenFeatureSIMD128 */,
+  Simd = 8 /* _BinaryenFeatureSIMD128 */,
```

Comment:
Similar to other abbreviations like `GC`. Should probably remain `SIMD`.

Reply:
Done!